core/block_dragger.js

```diff
@@ -10,61 +10,61 @@
  */
 'use strict';
 
-goog.provide('Blockly.BlockDragger');
-
-goog.require('Blockly.blockAnimations');
+goog.module('Blockly.BlockDragger');
+goog.module.declareLegacyNamespace();
+
+const BlockSvg = goog.requireType('Blockly.BlockSvg');
+const Coordinate = goog.require('Blockly.utils.Coordinate');
+const IBlockDragger = goog.require('Blockly.IBlockDragger');
+const IDragTarget = goog.requireType('Blockly.IDragTarget');
+const InsertionMarkerManager = goog.require('Blockly.InsertionMarkerManager');
+const WorkspaceSvg = goog.requireType('Blockly.WorkspaceSvg');
+const {BLOCK_DRAG, BLOCK_MOVE, fire, get, getGroup, setGroup} = goog.require('Blockly.Events');
+const {DEFAULT, Type, register} = goog.require('Blockly.registry');
+const {disconnectUiEffect, disconnectUiStop} = goog.require('Blockly.blockAnimations');
+const {startTextWidthCache, stopTextWidthCache} = goog.require('Blockly.utils.dom');
 /** @suppress {extraRequire} */
 goog.require('Blockly.constants');
```

Comment:
I think this can be removed now that this PR is in: https://github.com/google/blockly/pull/5140

Reply:
done.